Gaijinizer patch/Map041.txt

```diff
@@ -79,260 +81,268 @@ Remove mind corruption if(v[42] \>= 100) #Dargoth
-And one more thing, if you can not capture your
-soul...My body is self-destructing and
-disappearing.#MTLed
+The second is that in the event I could not absorb
+your soul... They would make my body self-destruct
+and I would disappear.#Ron
 > END STRING
 
 > BEGIN STRING
 私は最初、彼らの思惑通りに動くつもりでした。
 
 だって貴方は、本当に可愛らしい御方でしたもので…。
 > CONTEXT: Map041/events/5/pages/1/36/Dialogue/yuuma
-At first, I intended to move according to their
-expectations. Because you were a really pretty
-person ....#MTLed
+At first, I fully intended to comply with their
+wishes. I mean, you really were a lovely gentleman
+after all....#Ron
 > END STRING
 
 > BEGIN STRING
 しかし、あの施設にいた悪魔たちを見て…
 
 少々、気が変わりました。
 > CONTEXT: Map041/events/5/pages/1/40/Dialogue/yuuma
-However, I saw the demons who were in that
-facility...I changed my mind a little.#MTLed
+However, after seeing the demons in that facility
+my mind started to change little by little.#Ron
 > END STRING
 
 > BEGIN STRING
 あの施設の悪魔？
 
 ランパスや、バーゲストのことか？
 > CONTEXT: Map041/events/5/pages/1/44/Dialogue/kazuya
-The demon of that facility? Is it Lampas or
-barghest?#MTLed
+Demons in the facility? You mean the Barghests and
+Lampas?#Ron
 > END STRING
 
 > BEGIN STRING
 もはやご存じかと思いますが、どちらもヘカーテの従者…。
 
 ということは『ヘカーテ』自身も遊びに来ている…。
 > CONTEXT: Map041/events/5/pages/1/48/Dialogue/yuuma
-As you may already know, both of Hecate's
-followers .... That means "Hecate" is also coming
-to play ....#MTLed
+You may already know, but they're both Hectate's
+minions... That meant Hectate herself had come to
+play...#Ron
 > END STRING
 
 > BEGIN STRING
 つまり……貴方がどんなに必死に脱出しようとしても
 
 最終的には彼女に対峙することになってしまう…。
 > CONTEXT: Map041/events/5/pages/1/52/Dialogue/yuuma
-In other words...no matter how desperately you
-try to escape. Eventually you will face her ....#MTLed
+In other words... No matter how hard you tried to
+escape, you would eventually have to face her...#Ron
 > END STRING
 
 > BEGIN STRING
 そんな出来試合が待っているとも知らずに、
 必死に生きようとする貴方を見て…
 少し彼女の『シナリオ』を変えてやろうと思ったんです。
 > CONTEXT: Map041/events/5/pages/1/56/Dialogue/yuuma
-Without knowing that such a successful game is
-waiting, Seeing you trying to live desperately
-... I wanted to change her 'script' a little.#MTLed
+Seeing you desperately trying to live without
+knowing what kind of odds you were up against...
+I thought I'd try changing her "script".#Ron
 > END STRING
 
 > BEGIN STRING
 私はあの、昔から生命をオモチャ同然に扱う彼女が
 
 少々……好きではなかったものですから。
 > CONTEXT: Map041/events/5/pages/1/60/Dialogue/yuuma
-I treat her as a toy from the old days
-A little...because I did not like it.#MTLed
+From long ago she's treated lives as though they
+were no more than her playthings, and I...
+I slowly grew to dislike that.#Ron
 > END STRING
 
 > BEGIN STRING
 まさか………
 
 最後に僕を助けるために、わざと力を温存したのか？
 > CONTEXT: Map041/events/5/pages/1/64/Dialogue/kazuya
-No way ......... Lastly, did you intentionally
-save your power to help me?#MTLed
+No way...
+
+So in the end there did you intentionally retain
+your power so you could save me?#Ron
```

Comment:
"Retain" doesn't quite fit right here, it feels like she was holding back, or something similar.
>_**So in the end, did you hold back your power
just so you could save me?**_